pkg/server/api/trustdomain/v1/service.go

```diff
@@ -64,3 +80,75 @@ func (s *Service) BatchDeleteFederationRelationship(ctx context.Context, req *tr
 func (s *Service) RefreshBundle(ctx context.Context, req *trustdomainv1.RefreshBundleRequest) (*emptypb.Empty, error) {
 	return nil, status.Error(codes.Unimplemented, "unimplemented")
 }
+
+func fieldsFromRelationshipProto(proto *types.FederationRelationship, mask *types.FederationRelationshipMask) logrus.Fields {
```

Comment:
This standalone method should probably be placed after the methods for Service (which should all be grouped together).